deep-learning/README.md

```diff
@@ -19,15 +19,15 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
 2. [CS231n: Convolutional Neural Networks for Visual Recognition](http://cs231n.stanford.edu/)
 
 ### Roadmap to Deep Learning with Images
```

Comment:
Deep learning with images sounds weird. Instead, write something like "Roadmap to deep learning for perception/vision"